src/plugins/ffmemless/ffmemless.c

```diff
@@ -36,6 +36,7 @@
 int ffmemless_play(int effect_id, int device_file, int play)
 {
 	struct input_event event;
+	memset(&event, 0, sizeof(event));
```

Comment:
nit: does not matter here, but - unless one is forced to adhere to ancient standards - using initializer would be shorter and take member types into account.